### ISSUE TITLE
Error opening Delphi 10.3

### ISSUE THREAD

**Issue by zortatimo** [cnpack/cnwizards#92]
Hi,
I recently installed Delphi 10.3. The first thing I did was installing cnWizards, but I can't open Delphi with it installed, it gets stuck when loading cnWizards, showing several error messages.

This one is shown as soon as cnWizards starts to load.
![1](https://user-images.githubusercontent.com/25181155/54129382-e1bb4480-43ec-11e9-8965-12cdb50d2c58.png)

After the first Application Error, it shows this one:
![2](https://user-images.githubusercontent.com/25181155/54129386-e253db00-43ec-11e9-90fb-155e9acddb2e.png)

Then it shows the Tip of the Day as well as the Abstract error and keeps showing the abstract error, until I kill bds.exe.
![3](https://user-images.githubusercontent.com/25181155/54129387-e253db00-43ec-11e9-93ae-85d5073ceb4b.png)

I've tried the latest nightly build, but got the same results.
If there's some log or additional information I can provide, I'll be glad to.
Thanks.

**Comment by sebomozd** [comment]
I have the same issue.

**Comment by baka0815** [comment]
I don't have this issue with 10.3.1 but I'm getting #93.

**Comment by Sorien** [comment]
can be reproduced when you disable themes 

![image](https://user-images.githubusercontent.com/1270275/57283933-80dc7f80-70b0-11e9-8a70-96aa962ec4e2.png)

**Comment by shanzhashu** [comment]
Oh, thanks for the clue above. I guess there's a bug of AV when GetActiveThemeName from IDE if Theme Disabled. And the retrieving action is called at our initialization. I'll try to "catch" it.

**Comment by shanzhashu** [comment]
[CnWizards_D103R.dll.zip](https://github.com/cnpack/cnwizards/files/3153231/CnWizards_D103R.dll.zip)
I build a new DLL for 10.3 with adding the catch action, could anyone help to check whether the problem still exists or not?

**Comment by sebomozd** [comment]
Still exists, but i have 10.3.1 with themes enabled.

**Comment by shanzhashu** [comment]
Thanks sebomozd. then I'll build a debug version to record some logs in our DebugViewer. 

Can you try it to reproduce this problem again and save the logs in CnDebugViewer and paste here?
[CnWizards_D103R.dll.zip](https://github.com/cnpack/cnwizards/files/3155121/CnWizards_D103R.dll.zip)

**Comment by sebomozd** [comment]
Sure:
[CnWizards-logs.zip](https://github.com/cnpack/cnwizards/files/3156826/CnWizards-logs.zip)

**Comment by sebomozd** [comment]
Maybe the IDE is catching the exceptions?
Pasted some screenshots from debuger, maybe this can help. There are many exceptions related to duplicate keybindings. 
[Pictures.zip](https://github.com/cnpack/cnwizards/files/3158840/Pictures.zip)

**Comment by Sorien** [comment]
@shanzhashu my AV is fixed by your last changes ... it' seems it wasn't the same exception

**Comment by shanzhashu** [comment]
Yes, I also think they're different exceptions. However one fixed, good news.

**Comment by KngStr** [comment]
I have the same problem in win10 & 10.2.3
CnWizards_1.1.6.959 & CnWizards_1.1.7.975 both down.

**Comment by SantiPR** [comment]
Please, see this comment https://github.com/cnpack/cnwizards/issues/74#issuecomment-504572733

Hope this help.

**Comment by sebomozd** [comment]
> 
> 
> Please, see this comment [#74 (comment)](https://github.com/cnpack/cnwizards/issues/74#issuecomment-504572733)
> 
> Hope this help.

Thanks, it works now.

**Comment by KngStr** [comment]
> 
> 
> Please, see this comment [#74 (comment)](https://github.com/cnpack/cnwizards/issues/74#issuecomment-504572733)
> 
> Hope this help.

Thanks, it works now.

**Comment by haokucn** [comment]
[logs.zip](https://github.com/cnpack/cnwizards/files/4307676/logs.zip)
我也遇到这样得问题。

**Comment by haokucn** [comment]
Solution: uninstall Delphi and install it on disk D
uninstall  cnwizards and install it on disk D
